storage/storagedriver/overlayfs-driver.md

```diff
@@ -56,16 +62,9 @@ OverlayFS is supported if you meet the following prerequisites:
   re-create them later.
 
 
-## Configure Docker with the `overlay` or `overlay2` storage driver
-
-It is highly recommended that you use the `overlay2` driver if possible, rather
-than the `overlay` driver. The `overlay` driver is **not** supported for
-Docker EE.
+## Configure Docker with the `overlay2` storage driver
 
-To configure Docker to use the `overlay` storage driver your Docker host must be
-running version 3.18 of the Linux kernel (preferably newer) with the overlay
-kernel module loaded. For the `overlay2` driver, the version of your kernel must
-be 4.0 or newer.
+For the `overlay2` driver, the kernal version must be 4.0 or newer.
```

Comment:
@thaJeztah Is this also true for RHEL, given they backport stuff to 3.10 in 7.x branch? Perhaps, we add a qualifying statement on the OS?

Reply:
@ahh-docker RHEL uses an older kernel with 7.x branch. From the same page but different location:

> Note: If you use OverlayFS, use the overlay2 driver rather than the overlay driver, because it is more efficient in terms of inode utilization. To use the new driver, you need version 4.0 or higher of the Linux kernel, unless you are a Docker EE user on RHEL or CentOS, in which case you need version 3.10.0-514 or higher of the kernel and to follow some extra steps.

https://docs.docker.com/storage/storagedriver/overlayfs-driver/

Perhaps, we can reuse that wording here.

Reply:
@mataneja Given that we are deprecating the Overlay driver, I'd rather not reuse those words and use what @thaJeztah suggested instead.

Reply:
s/kernal/kernel/